source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+    |                        |                                  | application.                              |
+    +------------------------+----------------------------------+-------------------------------------------+
+    | ``--signal``           | ``#SBATCH --signal=USR1@300``    || Send the given signal to a job the       |
+    |                        |                                  | specified time (in seconds) seconds       |
+    |                        |                                  | before the job reaches its walltime. The  |
+    |                        |                                  | signal can be by name or by number (i.e.  |
+    |                        |                                  | both 10 and USR1 would send SIGUSR1).     |
+    |                        |                                  ||                                          |
+    |                        |                                  || Signaling a job can be used, for         |
+    |                        |                                  | example, to force a job to write a        |
+    |                        |                                  | checkpoint just before Slurm kills the    |
+    |                        |                                  | job (note that this option only sends the |
+    |                        |                                  | signal; the user must still make sure     |
+    |                        |                                  | their job script traps the signal and     |
+    |                        |                                  | handles it in the desired manner).        |
+    |                        |                                  ||                                          |
+    |                        |                                  || When used with ``sbatch``, the signal    |
+    |                        |                                  | can be prefixed by "B:" (e.g.             |
+    |                        |                                  | ``--signal=B:USR1@300``) to tell Slurm to |
+    |                        |                                  | signal only the batch shell; otherwise    |
+    |                        |                                  | all processes will be signaled.           |
+    +------------------------+----------------------------------+-------------------------------------------+
+
+
+Slurm Environment Variables
+---------------------------
 
-Module Commands
----------------
-Module Command line variables and descriptions
+Slurm reads a number of environment variables, many of which can provide the
+same information as the job options noted above. We recommend using the job
+options rather than environment variables to specify job options, as it allows
+you to have everything self-contained within the job submission script (rather
```

Comment:
Recommend comma instead of parens:

...self-contained within the job submission script, instead of having to remember what options you set for a given job.

Reply:
Updated in a new push.